examples/ganache/index.mjs

```diff
@@ -0,0 +1,41 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+import ethers from 'ethers';
+import ganache from 'ganache-core';
+const stdlib = loadStdlib();
+
+(async () => {
+  // OPTIONAL GANACHE CONFIG THAT CAN BE PASSED into ganache.provier()
+  const options = {
+    total_accounts: 1000,
+    gasPrice: '21000',
+    gasLimit: '5000000',
+    hardfork: "petersburg",
+    default_balance_ether: 7777777,
+    port: 7545,
+    ws: true,
+    vmErrorsonRPCResponse: true,
+    debug: true,
+    allowUnlimitedContractSize: true
+  };
```

Comment:
Can you make these options as minimal as possible and as recommended as possible? For example, I think we should suggest only having one account, because I *think* the faucet will be the first one. I also think we don't want people to have to know about the hardfork options. Whatever we put in the example will be copied over and over.

Reply:
ok. i only used high number of  accounts just to make sure alice connected to the right acct #999 when i printed logs and Alice was connected to ganacheAccount #999